resources/styles/book-content/questions.less

```diff
@@ -0,0 +1,13 @@
+.ost-chapter-review {
+  counter-reset: question;
```

Comment:
I talked to Derek about the pdf strategy for numbering and right now it's comparing apples and oranges since the items don't collate here. The numbering is ad-hoc for the pdf and varies widely from one pdf to another. So I think right now, we don't have a way of designing this particular problem with the pdf in mind. If we want to do so, we should ask W&N and figure out what they want, but the answer will be different for Bio and for HS Physics, and will apply to end of chapter collated items.